src/zvbi/sampling_par.c

```diff
@@ -448,10 +447,11 @@ _vbi_sampling_par_from_services_log
 				videostd_set |= par->videostd_set;
 		}
 
-		if (VBI_VIDEOSTD_SET_525_60 & videostd_set)
-			margin = 1.0e-6;
-		else
-			margin = 2.0e-6;
+		// double margin; /* Currently unused */
+		// if (VBI_VIDEOSTD_SET_525_60 & videostd_set)
```

Comment:
Why is this commented out?
zvbi is not our library. We need to leave it alone or we'll make our life more difficult when upgrading it.

Reply:
It just creates warnings :0 I'll try to resolve it the other way

Reply:
There is a way to suppress all the warnings from external libs in cmake file. Should I use it?